tests/Hello.java

```diff
@@ -1,5 +1,4 @@
 import java.lang.*;
-
```

Comment:
You might want to keep this empty line. The only reason I removed it is to make generated Hello.class being the same as the one in the source.